automation/taskcluster/decision_task.py

```diff
@@ -156,11 +156,7 @@ def gradle_module_task_name(module, gradle_task_name):
 
 def gradle_module_task(libs_tasks, module_info, is_release):
     module = module_info['name']
-    if is_release:
-        task_title = "{} - Build, test and upload to bintray".format(module)
-    else:
-        task_title = "{} - Build and test".format(module)
-    task = android_task(task_title, libs_tasks)
+    task = android_task("{} - Build and test".format(module), libs_tasks)
```

Comment:
Why not include some blurb about publishing for release builds?  It's cheap and might just save somebody some thought.  It also ties into my concern about CoT that I raise below.  Is _everything_ a release build now?  That doesn't match my model of how CoT should work.